src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -775,6 +778,97 @@ const CommandParametersPermissions& CommandRequestImpl::parameters_permissions()
   return parameters_permissions_;
 }
 
+void CommandRequestImpl::StartAwaitForInterface(
+    const HmiInterfaces::InterfaceID& interface_id) {
+  awaiting_response_interfaces_.insert(interface_id);
```

Comment:
@LuxoftAKutsan 
Looks like this variable can be accessed from multiple threads simultaneously. If so please consider mutex usage.